paragraph/translation/utils.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef PARAGRAPH_TRANSLATION_UTILS_H_
+#define PARAGRAPH_TRANSLATION_UTILS_H_
+
+#include <vector>
+
+#include "paragraph/graph/graph.h"
+
+namespace paragraph {
+
+// Helper functions for conversion bettwen consecutive processor id and
+// processor coordinates on a Mesh or Torus
+std::vector<uint64_t> ConsecutiveProcessorIdToMeshCoordinates(
```

Comment:
Call this either a cube (supersim does this) or a lattice (the proper term is an integer lattice).

Reply:
Renamed to grid as it is less confusing (Cube can be confused with HyperCube, Integer Lattice kinda long and looks synonymous to grid) in PR#24